src/uvs/univang.cpp

```diff
@@ -903,6 +903,8 @@ void uvsRestoreVanger(void){
 #endif
 }
 
+extern bool IsCurrentPassPointable;

```

Comment:
univang and units it's part of vangers (not a lib), we don't need to use "extern" here, it's just enough add this variable to header.